test/index.js

```diff
@@ -28,12 +28,12 @@ const runTest = function(url) {
         error: '>=2000'
       },
       psi: {
-        warn: '>=1500',
-        error: '>=2000'
+        warn: '>=2000',
+        error: '>=3500'
       },
       vc85: {
-        warn: '>=1500',
-        error: '>=2000'
+        warn: '>=3500',
+        error: '>=5000'
```

Comment:
We only pull from the gh api at build time so there should it shouldn't affect things, It's possible that the discourse api is causing the delay, but I didn't expect it difference to be so big. 

Pretty strange I checkout out `498530b30768bc1c26c5bfea65f479f9c313e232`, applied the rest of the changes on top of that and the tests (locally) were back to normal, trying to test it again on #86 but it looks like travis is dead.